data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone(mobile or home):
+- What city and country will you reside in during the summer?
```

Comment:
sounds good, giving them a choice so if they're comfortable we can arrange local meetings, if they don't - we don't